openpype/hosts/fusion/api/lib.py

```diff
@@ -68,8 +68,8 @@ def set_asset_framerange():
     asset_doc = get_current_project_asset()
     start = asset_doc["data"]["frameStart"]
     end = asset_doc["data"]["frameEnd"]
-    handle_start = asset_doc["data"]["handleStart"]
```

Comment:
I would say that these should be required.

Reply:
I changed from `asset_doc["data"]["handleStart"]` to `asset_doc["data"].get("handleStart")` so as `update_frame_range()` defaults handleStart/End to 0 if there is no data being given. Else the code would fail if there were no handleStart/End being set.

Reply:
From my point of view this is step back as `handleStart` and `handleEnd` should be required, but I'll let @jakubjezek001 and others to leave their opinions.

Reply:
This was a fix I made as the data from Kitsu didn't create any handleStart/End values for the asset. As update_frame_range() defaulted to 0 I thought an error in the code was made at some point.
But after doing some work on the Kitsu code it now creates all needed values.

As long as all other modules creates all required values this code could be removed.

Reply:
Hey @EmberLightVFX, it is unfortunate that Kitsu is not yet integrating `handleStart` and `handleEnd` but all our DCC integrations are in consistency with those attributes to be required. Solution would be to fix this from above. Kitsu integration should add those attributes at least in 0 form by default. 

It would be cool to merge it soon as I think this fix is very beneficial. We could allow this to be merged temporarily. Favorabely to be cross referenced to the Kitsu integration issue you've mentioned. Also, you can add some sort of hack or to-do comment above the change, please.